plugins/ad9371.c

```diff
@@ -707,7 +707,7 @@ static void rssi_update_labels(void)
 	rssi_update_label(obs_rssi, "voltage2", false);
 }
 
-static gboolean update_display(void)
+static gboolean update_display(gpointer foo)
```

Comment:
Not a major issue but could 'foo' be replaced with 'unused' or 'unusedParam'? (or other naming similar to this)